src/library/basetracktablemodel.h

```diff
@@ -18,9 +18,9 @@ class BaseTrackTableModel : public QAbstractTableModel, public TrackModel {
 
   public:
     explicit BaseTrackTableModel(
-            const char* settingsNamespace,
+            QObject* parent,
             TrackCollectionManager* const pTrackCollectionManager,
-            QObject* parent);
+            const char* settingsNamespace);
```

Comment:
If you like this can be a QString initialized by a QStringLiteral()
Since we store the value finally, you may use a value parameter + std::move() when passing.

Reply:
This would have to be changed in quite a few places, so I'd like to refrain from that for now